Unix/samples/Providers/Indication/util/util.h

```diff
@@ -246,7 +254,11 @@ MI_INLINE void Sem_Destroy(
 #else
     if (self->sem)
     {
+#ifdef __APPLE__
+    dispatch_release(self->sem);
```

Comment:
I'm not really familiar with the `dispatch_release` and related calls on Mac without investigation. Please have @yakman2020 (Bruce) review this, he's probably best.